Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
                     $dot = '.';
                     $id = '';
-                } else {
-                    $dot = '';
                 }
             }
 
-            if ($dot !== '.') {
-                throw new \Exception(sprintf('Could not get element id from %s Failing part: %s', $elementId, $subValue));
+            if (!empty($id)) {
```

Comment:
So if I understand well, it only fails when and only when the last iteration fails, am I right ?